src/main/java/org/jenkinsci/plugins/workflow/steps/durable_task/DurableTaskStep.java

```diff
@@ -94,7 +94,7 @@
  * <p>“Durable” in this context means that Jenkins makes an attempt to keep the external process running
  * even if either the Jenkins controller or an agent JVM is restarted.
  * Process standard output is directed to a file near the workspace, rather than holding a file handle open.
- * Whenever a Remoting connection between the two can be reëstablished,
+ * Whenever a Remoting connection between the two can be reestablished,
```

Comment:
Not a typo but rather a [diaeresis](https://en.wikipedia.org/wiki/Diaeresis_(diacritic)).